javascript/src/main/services/Courses/CourseService.js

```diff
@@ -0,0 +1,65 @@
+import { fetchWithToken } from "main/utils/fetch";
+ 
+const buildCreateCourse = (getToken, onSuccess, onError) => {
+  const func = async (name, quarter, instructorFirstName, instructorLastName, instructorEmail) => {
```

Comment:
The input should accept a course object instead
```suggestion
  const func = async (course) => {
```